stream_alert/rule_processor/handler.py

```diff
@@ -27,14 +28,17 @@
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import MetricLogger
 
-
+MAX_BATCH_COUNT = 500
+MAX_BATCH_SIZE = 4000 * 1000
```

Comment:
Can you add a comment about what this is? Is this a limit imposed by AWS?

Reply:
Good point, adding more clarificaiton